addons/events/fnc_keyHandlerDown.sqf

```diff
@@ -72,7 +72,7 @@ private _blockInput = false;
             _params pushBack + _keybindParams;
             _params pushBack _x;
 
-            _blockInput = ([_params call _code] param [0, false] isEqualTo true) || {_blockInput};
+            _blockInput = ([nil] apply {_params call _code} param [0, false] isEqualTo true) || {_blockInput};
```

Comment:
`0` would be faster than `nil` but would ofc define _x, doubt that would break anyones code?

Reply:
`_x` previously leaked into the user's code block alongside all the other locals like `_params`, `_code` and `_keybindParams` etc.
One could also use [_x] I suppose to keep the _x the same.
I just don't care about micro optimizations like that. The code is still a fifth of what it used to be before my rewrite and no one ever complained about it.
_x being nil was just the logical thing, as in "this variable I never defined is undefined", from the users perspective.